src/v2/providers/alerts/crashlytics.ts

```diff
@@ -69,42 +104,38 @@ export const velocityAlert = 'crashlytics.velocity';
 /** @internal */
 export const newAnrIssueAlert = 'crashlytics.newAnrIssue';
 
-/** Options */
+/**
+ * Configuration for crashlytics functions.
+ */
 export interface CrashlyticsOptions extends options.EventHandlerOptions {
   appId?: string;
 }
 
-/** Cloud Event Type */
-interface WithAlertTypeAndApp {
-  alertType: string;
-  appId: string;
-}
-export type CrashlyticsEvent<T> = CloudEvent<
-  FirebaseAlertData<T>,
-  WithAlertTypeAndApp
->;
+/** @internal */
```

Comment:
Please go back to inlining the callback type. It improves code complete in VS Code.

Reply:
Removed the handler type in the latest commit